diff -urN monotone-0.30.orig/m4/boost.m4 monotone-0.30/m4/boost.m4
--- monotone-0.30.orig/m4/boost.m4	2006-09-17 11:56:56.000000000 +0200
+++ monotone-0.30/m4/boost.m4	2006-09-29 12:15:51.000000000 +0200
@@ -57,9 +57,9 @@
 [AC_ARG_VAR([BOOST_SUFFIX],
             [Space-separated list of suffixes to try appending to the names
 	     of Boost libraries.  "none" means no suffix. The default is:
-	     "none -gcc -mipspro -mt -sunpro -sw -mgw -gcc-mt-s"])
+	     "none -gcc -mipspro -mt -sunpro -sw -mgw -gcc-mt -gcc-mt-s"])
 if test x"$BOOST_SUFFIX" = x; then
-  BOOST_SUFFIX="none -gcc -mipspro -mt -sunpro -sw -mgw -gcc-mt-s"
+  BOOST_SUFFIX="none -gcc -mipspro -mt -sunpro -sw -mgw -gcc-mt -gcc-mt-s"
 fi
 ])
 
